src/Cardano/BM/Configuration/Model.lhs

```diff
@@ -164,11 +164,20 @@ or, in case no such configuration exists, return the default scribes to use.
 getScribes :: Configuration -> LoggerName -> IO [ScribeId]
 getScribes configuration name =
     withMVar (getCG configuration) $ \cg -> do
+        let defs = cgDefScribes cg
         let outs = HM.lookup name (cgMapScribe cg)
-        case outs of
-            Nothing -> do
-                return (cgDefScribes cg)
-            Just os -> return $ os
+        let find_s lname = case outs of
+                Nothing -> do
+                    case dropToDot lname of
```

Comment:
if we can't find a configuration for this name, we drop the last entry in the name and continue the search.